File: comms/email/delete-email/delete-email.go
package example

import (
    context "context"

    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "<username>",
            "<password>",
        ),
    )
    client.Emails.Delete(
        context.TODO(),
        "comms_email_01h9krwprkeee8fzqspvwy6nq8",
    )
}

File: comms/email/delete-email/delete-email.cs
using TwilioComms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioCommsClient(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.Emails.DeleteAsync(
            "comms_email_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}
